build.gradle

```diff
@@ -32,32 +33,38 @@ buildscript {
 
 subprojects {
   afterEvaluate { project ->
-    task sourcesJar(type: Jar) {
-      from android.sourceSets.main.java.srcDirs
-      classifier = 'sources'
-    }
 
-    task javadoc(type: Javadoc) {
-      source = android.sourceSets.main.java.srcDirs
-      classpath += project.files(android.getBootClasspath().join(File.pathSeparator))
-      classpath += fileTree(dir: "$buildDir/intermediates/exploded-aar/", include: "**/classes.jar")
-      classpath += configurations.compile
-      // Exclude generated files
-      exclude '**/BuildConfig.java'
-      exclude '**/R.java'
-    }
+    // These two projects are JARed and DOCed using their own build.gradle files
+    if (!project.projectDir.absolutePath.contains("backbone") && !project.projectDir.absolutePath.contains("skin")) {
```

Comment:
Also, @liujoshua what is the purpose of creating the JAR task?  My guess  would be to deploy it to bintray, but I wasn't completely sure